client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -32,6 +31,11 @@ export function getEnhancedTasks(
 				case 'setup_newsletter':
 					taskData = {
 						title: translate( 'Set up Newsletter' ),
+						keepActive: true,
+						actionDispatch: () =>
```

Comment:
At this point we are using actionDispatch for almost every case, only 'first-post-published' and 'links-added' are using actionUrl to perform the redirect, currently they don't exist on the StepPath type. Do you think we should add them there and use only actionDispatch with the goToStep method or we should keep using actionUrl

Reply:
@Sharpirate - First post published is outside the stepper path as you noted, so we wouldn't use one of the stepper methods. That said, we could plausibly use actionDispatch, and just refactor downstream to use that method rather than actionUrl. There may be some advantage to just using that for all tasks / links. But I'd probably consider that a separate PR to remove actionUrl here and from the ChecklistItem component.

For the Links Added, I think that may be updated to use actionDispatch when we work through [this issue ](https://github.com/Automattic/wp-calypso/issues/67430https://github.com/Automattic/wp-calypso/issues/67430), which I think is one you'll be taking...